src/lib/stitching/kaws/__tests__/__snapshots__/schema.test.ts.snap

```diff
@@ -4,7 +4,6 @@ exports[`creates an SDL 1`] = `
 "# Object representing a collection page
 type MarketingCollection {
   id: ID!
-  internalID: ID!
```

Comment:
Why does it go away in this snapshot?

Reply:
This is a snapshot of the transformed but not merged KAWS schema (not MP).